cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,72 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('should can change rows count', () => {
+    cy.selectCountRows('5 rows');
+    cy.get('[aria-label="rows per page"]').should('contain', '5 rows');
+  });
+
+  it('should add a new worker', () => {
```

Comment:
The test 'should add a new worker' lacks assertions to verify that the worker was actually added. Consider adding assertions to check the presence of the new worker in the table.